R/post_ts.R

```diff
@@ -54,10 +54,8 @@ post_ts = function(files, on_exists=c("stop", "skip", "replace", "merge","archiv
     ts_id <- locate_ts(var_src=ts_path$var_src, site_name=ts_path$site_name, by="either")
     if (is.na(ts_id)) {
       ts_id <- item_create(site_root, title=ts_path$ts_name)$id
+      repair_ts(ts_path$var_src, ts_path$site_name)
```

Comment:
note that this does seem to be necessary when the initial item is created. Was left off last PR and showed up w/ testing.